iceoryx_posh/testing/include/iceoryx_posh/testing/mocks/chunk_mock.hpp

```diff
@@ -47,7 +46,7 @@ class ChunkMock
         auto& chunkSettings = chunkSettingsResult.value();
         auto chunkSize = chunkSettings.requiredChunkSize();
 
-        m_rawMemory = static_cast<uint8_t*>(iox::cxx::alignedAlloc(alignof(iox::mepoo::ChunkHeader), chunkSize));
+        m_rawMemory = static_cast<uint8_t*>(iox::alignedAlloc(alignof(iox::mepoo::ChunkHeader), chunkSize));
```

Comment:
@dkroenke I thought for tests and mocks we don't really care about implicit includes. I can still add those if you think it's important.

Reply:
@mossmaurice Beside the better readability it makes a possible refactoring in future easier when header files get reworked and the build does not break at unexpected places. Due to the amount of includes here it would be a good idea.

Reply:
@dkroenke That's correct. I'll add them. For the `into.hpp` it doesn't make much sense to me however. All the tests don't use the generic `into` but the specialized template from `std_string_support.hpp`. IMHO a further include of `into` would actually be misleading.

Reply:
@mossmaurice Ok that's fine, resolved.